static/redfish.yaml

```diff
@@ -1022,6 +1094,8 @@ paths:
   /TaskService/Oem/Tasks/{identifier}:
     x-swagger-router-controller: task-service
     get:
+      x-privileges: [ 'Login' ]
```

Comment:
should we be building up a list of privileges to know what we've used (and should re-use) in the developer's guide or something? tying to figure out how to track these pieces to know how/when to re-use them and assert our common conventions for when we need to extend this later.